apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -203,16 +227,14 @@ const CustomizableCurriculumCatalogCard = ({
               __useDeprecatedTag
               color={Button.ButtonColor.neutralDark}
               type="button"
-              href={pathToCourse}
+              onClick={handleClickLearnMore}
```

Comment:
The `__useDeprecatedTag` here will render this as an [anchor tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a), which shouldn't have an onClick.

You could remove the useDeprecatedTag to force it to render as a `button` instead of an `a` tag. That allows for the event to be processed, but then there are [accessibility concerns](https://www.washington.edu/accesstech/websites/links-buttons/) because it is really a link that takes you to a new page. Perhaps you could add role=”link” prop? But you might need to add something to the `Button.jsx` component to spread/add that prop into the <Tag>. If you go that route, could you also check that you can open this link in a new tab via keyboard (on a Mac, Cmd+Click or Cmd+Enter)?
 
https://github.com/code-dot-org/code-dot-org/blob/1a6bb7ef3583aa0a9de0f887915b1a277f1ea69f/apps/src/templates/Button.jsx#L175

Reply:
This changed has been removed since Dani and I spoke and there's no need for the CURRICULUM_CATALOG_LEARN_MORE_CLICKED_EVENT event since Juan is about to update the expanded card and we already get events when a user visits a Unit Overview or Course Overview page (so we would theoretically know if a user is on the catalog page then visits one of those pages that they probably clicked 'Learn more').